drivers/metadata/informationschema/metadata.go

```diff
@@ -158,6 +160,14 @@ func WithCurrentSchema(expr string) metadata.ReaderOption {
 	}
 }
 
+// WithDataTypeFormatter function to build updated string represenation of data type
+// from Column
+func WithDataTypeFormatter(form func(metadata.Column) string) metadata.ReaderOption {
```

Comment:
Nit: replace `form` with either `f` or `formatter`, `form` is a bit misleading as it is a regular word with a different meaning.